snaps-registry.json

```diff
@@ -44,7 +44,8 @@
         "audits": [
           {
             "auditor": "OtterSec",
-            "report": "https://ottersec.notion.site/Sampled-Public-Audit-Reports-a296e98838aa4fdb8f3b192663400772?p=b25b9f3a52ab4821914f379ff0d3ec62&pm=s"
+            "report": "https://ottersec.notion.site/Sampled-Public-Audit-Reports-a296e98838aa4fdb8f3b192663400772?p=b25b9f3a52ab4821914f379ff0d3ec62&pm=s",
+            "address": "0x2bE127806140dFd6c4DC1cAA0e828411B77E4BbF"
```

Comment:
Since auditors can do multiple audits I think the address should be a separate top level section.

```jsonc
{
  "verifiedSnaps": { /* ... */ },
  "auditors": {
     "OtterSec": {
        "address": "0x2bE127806140dFd6c4DC1cAA0e828411B77E4BbF"
     }
  }
}
```

Reply:
Yes indeed, thanks for the feedback.